bftengine/src/bftengine/RequestHandler.cpp

```diff
@@ -72,6 +80,9 @@ void RequestHandler::execute(IRequestsHandler::ExecutionRequestsQueue& requests,
     } else if (req.flags & MsgFlag::RECONFIG_FLAG) {
       ReconfigurationRequest rreq;
       deserialize(std::vector<std::uint8_t>(req.request, req.request + req.requestSize), rreq);
+      LOG_INFO(
```

Comment:
same

Reply:
Shouldn't reconfiguration requests be rare?

Reply:
it should, we can leave it in INFO

Reply:
nevermind, already changed to debug